fudge-ratio/src/main/webapp/script.js

```diff
@@ -71,6 +81,57 @@ buttonClear.addEventListener("click", () => {
   cleanSessionState();
 });
 
+buttonAddHour.addEventListener("click", () => {
+    durationHour += 1;
+    durationText.textContent = padd(durationHour) + ":" + padd(durationMin) + ":" + padd(durationSec);
```

Comment:
This line seems very repetitive, maybe create a function, so we dont need to change 6 lines if something changes?